app/views/settings/recruitment/sources/_sources_edit.html.slim

```diff
@@ -0,0 +1,69 @@
+/# locals: (tenant:, all_sources:)
+
+ruby:
+  order_field_collection =
+    all_sources + [CandidateSource.new]
+
+#rercruitment_sources
```

Comment:
Then form works incorrectly after the update. I don't fully understand the reason

Reply:
i used `settings_form` anchor in my pr for updating and saving email templates and thought it would be good to reuse the same behaviour here. 
I'm ok with your solution
please explain what works incorrectly with my suggestion